migrations/sql/V2019.06.26.10.30__add_code_required_report_tables.sql

```diff
@@ -0,0 +1,154 @@
+    FOREIGN KEY (mine_report_submission_id) REFERENCES mine_report_submission(mine_report_submission_id) DEFERRABLE INITIALLY DEFERRED
+);
+
+COMMENT ON TABLE mine_report_document_xref is 'Links a mine document to report submission.';
+ALTER TABLE mine_report_document_xref OWNER TO mds;
+
+CREATE TABLE IF NOT EXISTS mine_report_comment (
+    mine_report_comment_id           serial                                  NOT NULL PRIMARY KEY,
+    mine_report_comment_guid         uuid                                    NOT NULL            ,
+    mine_report_id                   integer                                 NOT NULL            ,
+    mine_report_submission_id        integer                                                     ,
+    comment                          varchar(300)                            NOT NULL            ,
+    minespace_user_id                integer                                                     ,
+    core_user_id                     integer                                                     ,
+    comment_visibility               boolean                                 NOT NULL            ,
+    create_user                      character varying(60)                   NOT NULL            ,
+    create_timestamp                 timestamp with time zone DEFAULT now()  NOT NULL            ,
+    update_user                      character varying(60)                   NOT NULL            ,
+    update_timestamp                 timestamp with time zone DEFAULT now()  NOT NULL            ,
+
+    FOREIGN KEY (mine_report_id) REFERENCES mine_report(mine_report_id) DEFERRABLE INITIALLY DEFERRED,
+    FOREIGN KEY (mine_report_submission_id) REFERENCES mine_report_submission(mine_report_submission_id) DEFERRABLE INITIALLY DEFERRED,
+    FOREIGN KEY (core_user_id) REFERENCES core_user(core_user_id) DEFERRABLE INITIALLY DEFERRED,
+    FOREIGN KEY (minespace_user_id) REFERENCES minespace_user(user_id) DEFERRABLE INITIALLY DEFERRED
+);
+
+COMMENT ON TABLE mine_report_comment is 'A comment on a report or report submission from a proponent or core user.';
+ALTER TABLE mine_report_comment OWNER TO mds;
+
+CREATE TABLE mine_report_category (
+    mine_report_category    character varying(3)                     PRIMARY KEY,
+    description              character varying(100)                  NOT NULL   ,
+    display_order            smallint                                NOT NULL   ,
+    active_ind               boolean DEFAULT true                    NOT NULL   ,
+    create_user              character varying(60)                   NOT NULL   ,
+    create_timestamp         timestamp with time zone DEFAULT now()  NOT NULL   ,
+    update_user              character varying(60)                   NOT NULL   ,
+    update_timestamp         timestamp with time zone DEFAULT now()  NOT NULL
+);
+
+COMMENT ON TABLE mine_report_category is 'Categories available for mine reports..';
```

Comment:
Can you please provide a couple of examples of "Categories" in the table comment?

Reply:
In the comment or here? This table is more for the business to tag reports such as geotechnical, health and safety and so on.

Reply:
I think just adding some examples within the table description...

"Categories available for mine reports, which are used to tag the reports with business terms. For example, Geotechnical, Health and Safety, etc."

...something like this.

Reply:
I added those to the comment.